test/utils/utils.go

```diff
@@ -0,0 +1,32 @@
+package utils
```

Comment:
In the go world, people usually try to avoid having utils packages https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common